src/main.rs

```diff
@@ -132,7 +151,7 @@ async fn entry() -> Result<()> {
     }
 
     // Load options
-    let mut opts = Options::from_iter(args.iter());
+    let mut opts = Options::parse_from(args.iter());
```

Comment:
```suggestion
    let mut opts = Options::parse_from(args);
```

It's more efficient to pass it directly since [`clap::Parser`](https://docs.rs/clap/latest/clap/trait.Parser.html) expects `OsString`, so passing `args.iter()` would cause them to be `clone`d.